tests/Unit/Domain/Services/TalkRating/OneToTenRatingTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Test\Unit\Domain\Services\TalkRating;
+
+use Mockery;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+use OpenCFP\Domain\Model\TalkMeta;
+use OpenCFP\Domain\Services\Authentication;
+use OpenCFP\Domain\Services\TalkRating\OneToTenRating;
+
+/**
+ * @covers \OpenCFP\Domain\Services\TalkRating\OneToTenRating
+ */
+class OneToTenRatingTest extends \PHPUnit\Framework\TestCase
+{
+    use MockeryPHPUnitIntegration;
+
+    /**
+     * @dataProvider ratingProvider
+     */
+    public function testValidRatings($rating, $valid)
+    {
+        $mockAuth = Mockery::mock(Authentication::class);
+        $mockAuth->shouldReceive('userId');
+        $metaMock    = Mockery::mock(TalkMeta::class);
+        $oneToTen    = new OneToTenRating($metaMock, $mockAuth);
+        $this->assertSame($valid, $oneToTen->isValidRating($rating));
+    }
+
+    public function testGetRatingNameReturnsOneToTen()
+    {
+        $mockAuth = Mockery::mock(Authentication::class);
+        $mockAuth->shouldReceive('userId');
+        $metaMock    = Mockery::mock(TalkMeta::class);
+        $oneToTen    = new OneToTenRating($metaMock, $mockAuth);
+        $this->assertSame('onetoten', $oneToTen->getRatingName());
+    }
+
+    public function ratingProvider()
```

Comment:
What do you think about splitting up the tests and the data providers?

* one for valid rating
* one for invalid rating

?